src/cljs/replumb/core.cljs

```diff
@@ -203,11 +218,18 @@
   `read-file-fn!`, an asynchronous 2-arity function with signature
   `[file-path src-cb]` where src-cb is itself a function `(fn [source]
   ...)` that needs to be called with the file content as string (`nil`
-  if no file is found)."
+  if no file is found).
+
+  The 3-arity function receives additionally a third parameter `write-file-fn!`,
+  a synchronous 2-arity function of signature `[file-path data]`."
   ([load-fn!]
    {:target :nodejs
     :load-fn! load-fn!})
   ([src-paths read-file-fn!]
-   {:target :nodejs
-    :read-file-fn! read-file-fn!
-    :src-paths src-paths}))
+   (nodejs-options src-paths read-file-fn! nil))
+  ([src-paths read-file-fn! write-file-fn!]
+   (merge {:target :nodejs
+           :read-file-fn! read-file-fn!
+           :src-paths src-paths}
+          (when write-file-fn!
```

Comment:
This check is not necessary as a `(get ... :write-file-fn!)` would return `nil` anyways